app/src/BackButton.tsx

```diff
@@ -0,0 +1,11 @@
+import { useNavigate } from "react-router-dom";
```

Comment:
Thank you for adding the button for users to click back on previous page, this is great work, we can test this later 🧪